docs/source/user-guide/tutorials/building-conda-packages.rst

```diff
@@ -48,14 +48,15 @@ version of Visual Studio (VS):
 
 * Python 2.7 packages with Visual Studio 2008
 * Python 3.4 packages with VS 2010
-* Python 3.5 packages with VS 2015
-* Python 3.6 packages with VS 2015
+* Python 3.5 packages with VS 2015, (default) 2017
+* Python 3.6 packages with VS 2015, (default) 2017
+* Python 3.7 packages with VS 2015, (default) 2017
 
 Using these versions of VS to build packages for each of these
 versions of Python is also the practice used for the official
 python.org builds of Python. Currently VS 2008 and VS 2010 are
-available only through resellers, while VS 2015 can be purchased
-online from Microsoft.
+available only through resellers, while VS 2015 and VS 2017 can
+be purchased online from Microsoft.
```

Comment:
VS2015 and 2017 have community editions that are fully functional.  I'm not sure if we want to link to those here.